tests/test_client.py

```diff
@@ -26,46 +25,32 @@ def test_init(api_client: ApiClient):
     assert api_client
 
 
-def test_timeout_default(httpx_mock: HTTPXMock):
-    httpx_mock.add_response(
-        match_extensions={
-            "timeout": {
-                "connect": DEFAULT_TIMEOUT_CONFIG.connect,
-                "read": DEFAULT_TIMEOUT_CONFIG.read,
-                "write": DEFAULT_TIMEOUT_CONFIG.write,
-                "pool": DEFAULT_TIMEOUT_CONFIG.pool,
-            }
-        },
-        status_code=httpx.codes.BAD_GATEWAY,
+def test_base_url_override(respx_mock: respx.mock):
+    respx_mock.get(re.compile(r"^https://test/repos/foo/bar/import/issues")).mock(
```

Comment:
Do you need `re.compile` here? It should work without it.

Reply:
Yes, it's really needed, otherwise `respx` can't know that it's a regular expression and not a normal string to be matched literally.

The alternative is to use the `url__regex` parameter like in 1b45fec, but I'm not sure which is better, because `re.compile` tells PyCharm that it's a regex and you get the appropriate highlighting :)

Reply:
I think `startswith` is even better (f8ea96b) ;-)

Reply:
You can remove `url__startswith`. `respx` works differently compared to `pytest_httpx`. `respx` matches all URLs started from `https://test/repos/foo/bar/import/issues`.